blank.json

```diff
@@ -1,6 +1,5 @@
 {
     "region": 13,
-    "altitude": 0,
```

Comment:
Presumably `altitude` was used in the model at some point (to determine external temperature), but no longer seems to be part of the calculations.